docs/Resumption/index.md

```diff
@@ -0,0 +1,58 @@
+# Application Data Resumption
+
+Data resumption is used when an application requests to restore data used in the previous ignition cycle after an unexpected disconnect. Resumption starts during the `RegisterAppInterface` request message processing. The application must include the hashID parameter in the request for SDL Core to recognize a previous connection and attempt to restore as much of the previous HMI state as possible. If the hashID received from mobile is correct, SDL Core will restore the following data to the HMI.
+
+- Menu Items (AddCommands)
+- Sub Menus
+- Choice Sets
+- Persistent Files and Images
+- Global Properties
+- Button Subscriptions
+- App Service Subscriptions
+- Vehicle Data Subscriptions
+- Remote Control Module Consent
+
+For data resumption purposes, SDL Core must store application-related data such as menu commands, application global properties, voice recognition data, and subscriptions for previous ignition cycles after an unexpected disconnect or Ignition Off. 
+
+!!! note
+If your SDL Core integration is configured to use a resumption data base, on every fourth ignition cycle, SDL clears all corresponding application-related data used for resumption.
+
+smartDeviceLink.ini example
+```
+; Resumption ctrl uses JSON if UseDBForResumption=false for store data otherwise uses DB
+UseDBForResumption = false
+```
+
+This does not apply to the default configuration to use the app_info.dat JSON resumption file.
+!!!
+
+HMI must store the VR grammars compiled for applications that are unregistered by an unexpected disconnect or ignition Off. During data resumption, the HMI may also have to resume the previous audio source. Refer to `BasicCommunication.OnResumeAudioSource`.
```

Comment:
```suggestion
HMI must store the VR grammars compiled for applications that are unregistered by an unexpected disconnect or Ignition Off. During data resumption, the HMI may also have to resume the previous audio source. Refer to `BasicCommunication.OnResumeAudioSource`.
```

Reply:
With this change I would lowercase the following Off as well

Reply:
I went with "ignition off"